skimage/feature/tests/test_hog.py

```diff
@@ -287,3 +287,16 @@ def test_hog_output_equivariance_channel_axis(channel_axis):
         hog_fact = feature.hog(np.roll(img, n, axis=channel_axis),
                                channel_axis=channel_axis, block_norm='L1')
         assert_almost_equal(hog_ref, hog_fact)
+
+
+def test_hog_small_image():
+    """
+    tests that an exception is thrown instead of an empty array returned when
+    the image is smaller than it should be for the given parameters
+    """
+    img = np.zeros((24, 193))
+    feature.hog(img)
+
+    img = np.zeros((15, 193))
```

Comment:
I think it would be more future-compatible to explicitly specify the values of `pixels_per_cell` and `cells_per_block` parameters in this test.

Reply:
That makes sense I have updated this accordingly.